src/CabanaPD_Solver.hpp

```diff
@@ -133,6 +137,12 @@ class SolverElastic
                                     TemperatureDependent>::value )
             force_model.update( particles->sliceTemperature() );
 
+        // Define heat transfer if being used.
+        if constexpr ( std::is_same<typename force_model_type::thermal_type,
+                                    DynamicTemperature>::value )
+            heat_transfer = std::make_shared<heat_transfer_type>(
```

Comment:
What does this line do exactly? What does "define heat transfer" actually mean?

Reply:
Could say "construct heat transfer" instead

Reply:
That may be better.